pkg/controller/controller.go

```diff
@@ -37,7 +37,7 @@ import (
 )
 
 const (
-	tprName = "etcd-cluster.coreos.com"
+	TPRName = "etcd-cluster.coreos.com"
```

Comment:
Client needs to know the TPR name to operator. see your comment here https://github.com/coreos/etcd-operator/pull/643#discussion_r95631853